### ISSUE TITLE
求增加使用MOSS模型，因为MOSS是完全免费的

### ISSUE THREAD

**Issue by lhtpluto** [l15y/wenda#280]
求增加使用MOSS模型，因为MOSS是完全免费的

**Comment by l15y** [comment]
已支持

**Comment by lhtpluto** [comment]
> 已支持

非常感谢


运行时遇到错误

The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:106068 for open-end generation.
错误 Both `max_new_tokens` and `max_length` have been set but they serve the same purpose -- setting a limit to the generated output length. Remove one of those arguments. Please refer to the documentation for more information. (https://huggingface.co/docs/transformers/main/en/main_classes/text_generation)


我尝试了 从https://github.com/huggingface/transformers 上重新安装了transformers（https://github.com/huggingface/transformers/issues/21369），进展是能运行程序了，但每轮对话任然提示这个错误
